libs/@guardian/react-crossword/src/components/Grid.tsx

```diff
@@ -134,6 +134,25 @@ export const Grid = () => {
 
 	const [cheatMode, cheatStyles] = useCheatMode(gridRef);
 
+	const handleCurrentCellClick = useCallback(
+		(cell: CellType) => {
+			if (
+				cell.x === currentCell.x &&
+				cell.y === currentCell.y &&
+				cell.group?.length
+			) {
+				const otherEntry = cell.group.find(
+					(entryId) => entryId !== currentEntryId,
+				);
+				if (otherEntry) {
+					setCurrentEntryId(otherEntry);
+					return;
+				}
```

Comment:
Just on the naming. We are finding `otherEntry` but then using it to `setCurrentEntryId` ?

```suggestion
				const otherEntry = cell.group.find(
					(entryId) => entryId !== currentEntryId,
				);
				if (otherEntry) {
					setCurrentEntry(otherEntry);
					return;
				}
```

Reply:
Good point. I renamed to `otherEntryId`